FluentSharp_Fork.WatiN

```diff
@@ -1 +1 @@
-Subproject commit 131b0268b999409f5de12f576217525ebb1f1370
+Subproject commit 5c574ab983ad6671a42f9d4db85ea5007def744a
```

Comment:
a.stanimir@CP8101H8J0T1WGY ~/Documents/GitHub/FluentSharp/FluentSharp_Fork.SharpDevelopEditor ((70cd79d...))
$ git status
HEAD detached at 70cd79d
nothing to commit, working directory clean

I remembered that I had to update also submodules during the merge.